src/HogWildTrainer.scala

```diff
@@ -1,22 +0,0 @@
-
-import cc.factorie.optimize.{ GradientOptimizer, Trainer, Example }
-import cc.factorie.la.SmartGradientAccumulator
-import cc.factorie.util.{ LocalDoubleAccumulator, Threading }
-import cc.factorie.model.WeightsSet
-
-class HogWildTrainer(val weightsSet: WeightsSet, val optimizer: GradientOptimizer, val nThreads: Int = Runtime.getRuntime.availableProcessors(), val maxIterations: Int = 3)
```

Comment:
HogWild seems to be a design paradigm for SGD and related optimizers that improves computing efficiency. [See this paper](https://people.eecs.berkeley.edu/~brecht/papers/hogwildTR.pdf).